lib/active_record/connection_adapters/odbc_adapter.rb

```diff
@@ -1564,7 +1555,8 @@ def select(sql, name) # :nodoc:
           end
 
           ## Adjust Ruby string encoding, if requested
-          if rRows && enc=@connection_options[:ruby_string_encoding]
+          if enc=@connection_options[:ruby_string_encoding]
```

Comment:
Might want to enclose this in parentheses to indicate it's a statement and not part of the conditional.